indico/modules/events/logs/models/entries.py

```diff
@@ -96,6 +97,11 @@ class EventLogEntry(db.Model):
         JSON,
         nullable=False
     )
+    #: Non-displayable data
+    internal_data = db.Column(
+        JSON,
```

Comment:
Let's use JSONB here so we can use an index. JSON is stored as opaque text and won't be indexed. The only reason `data` is still JSON is that converting it would have taken ages in our own database.